File: app/ecs/systems/CollideDetectorSystem.lua
local ____exports = {}
local ____ECS = require("ecs.core.ECS")
local ecs = ____ECS.ecs
function ____exports.CollideDetectorSystem()
    local tmp = vmath.vector3()
    return ecs.systems.create(
        {"CollidableComponent", "GO"},
        {
            update = function(____, e)
            end,
            on_entity_added = function(self, entity)
            end
        }
    )
end
return ____exports

File: src/ecs/systems/CollideDetectorSystem.ts
import { ecs } from "../core/ECS";
export function CollideDetectorSystem() {
    const tmp = vmath.vector3();
    return ecs.systems.create(['CollidableComponent', 'GO'], {
        update: (e) => {

        },

        on_entity_added(entity) {

        },
    });
}